tests/Aspire.Hosting.Redis.Tests/Aspire.Hosting.Redis.Tests.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsAspireHost>true</IsAspireHost>
+    <!--
+      CS8002: Referenced assembly does not have a strong name. MongoDB.Driver package is unsigned, we ignore that warning on purpose
+    -->
+    <NoWarn>$(NoWarn);CS8002</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Aspire.Hosting.AppHost\Aspire.Hosting.AppHost.csproj" IsAspireProjectResource="false" />
+    <ProjectReference Include="..\..\src\Aspire.Hosting.Redis\Aspire.Hosting.Redis.csproj" IsAspireProjectResource="false" />
+    <ProjectReference Include="..\..\src\Components\Aspire.StackExchange.Redis\Aspire.StackExchange.Redis.csproj" />
+
+    <PackageReference Include="Microsoft.Extensions.Http.Resilience" />
+    <PackageReference Include="Newtonsoft.Json.Schema" />
+
+    <Compile Include="$(TestsSharedDir)Logging\*.cs" LinkBase="shared/Logging" />
+
+    <Compile Include="..\Aspire.Hosting.Tests\Utils\ArgumentEvaluator.cs" Link="Utils\ArgumentEvaluator.cs" />
+    <Compile Include="..\Aspire.Hosting.Tests\Utils\EnvironmentVariableEvaluator.cs" Link="Utils\EnvironmentVariableEvaluator.cs" />
+    <Compile Include="..\Aspire.Hosting.Tests\Utils\FileUtil.cs" Link="Utils\FileUtil.cs" />
+    <Compile Include="..\Aspire.Hosting.Tests\Utils\ManifestUtils.cs" Link="Utils\ManifestUtils.cs" />
+    <Compile Include="..\Aspire.Hosting.Tests\Utils\TestDistributedApplicationBuilder.cs" Link="Utils\TestDistributedApplicationBuilder.cs" />
+    <Compile Include="..\Aspire.Hosting.Tests\XunitAttributes.cs" Link="XunitAttributes.cs" />
+    
+    <Content Include="$(RepoRoot)src\Schema\aspire-8.0.json" Link="Schema\aspire-8.0.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
```

Comment:
Is this needed?

Reply:
I don't think so, I think this might be copy-pasta. We have schema tests in Aspire.Hosting.Tests that does need this but those tests aren't really Redis specific (although they do call AddRedis to verify that the manifest output complies to the schema).

If we wanted to move those schema tests over then we would need this file.